addons/mail/static/src/owl/components/activity/activity.scss

```
@@ -0,0 +1,174 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_Activity {
+    display: flex;
+    flex: 0 0 auto;
+    padding: 5px;
```

When the existing value is not on the list (example here 5px, we need to decide if we want to make it 4px or 8px). -> probably 4px in this case.

```suggestion
    padding: map-get($spacers, 2);
```